example/app/src/main/java/example/pilgrimsdk/kits/mparticle/com/pilgrimsdkmparticle/activities/MainActivity.kt

```diff
@@ -79,6 +78,10 @@ class MainActivity : AppCompatActivity() {
             if (intent.action!!.startsWith(MParticle.ServiceProviders.BROADCAST_ACTIVE)) {
                 //make a direct PilgrimSDK API call, or set a boolean field that you can check elsewhere
                 if (context.isLocationPermissionGranted()) {
+                    // NOTE: If you're setting persisteng logging off in mParticle's dashboard
+                    // please uncomment this, so you can see stuff happening in the next activity
+                    // PilgrimSdk.get().setEnablePersistentLogs(true)
```

Comment:
this is correct, though if its not also set in the initialization, it may or may not miss logs that are logged after the app process dies and launches back up. Think its fine just wanted to call out the app lifecycle issues that might lead to slightly unexpected behavior

Reply:
I'll leave the comment, they should enable it through the dashboard